public/app/features/plugins/admin/components/PluginDetailsPanel.tsx

```diff
@@ -99,8 +99,14 @@ export function PluginDetailsPanel(props: Props): React.ReactElement | null {
                     <Trans i18nKey="plugins.details.labels.repository">Repository</Trans>
                   </LinkButton>
                 )}
-                {plugin.raiseAnIssueUrl && (
-                  <LinkButton href={plugin.raiseAnIssueUrl} variant="secondary" fill="solid" icon="bug" target="_blank">
+                {plugin.details?.raiseAnIssueUrl && (
```

Comment:
It would be a good idea to extend https://github.com/grafana/grafana/blob/80af3703d216fba5adfb09a9319bf19181d3d8bb/public/app/features/plugins/admin/components/PluginDetailsPanel.test.tsx to include the use case of `raiseAnIssueUrl` being present